modelscope/models/multi_modal/stable_diffusion/stable_diffusion.py

```diff
@@ -34,6 +36,7 @@ def __init__(self, model_dir, *args, **kwargs):
         """
         super().__init__(model_dir, *args, **kwargs)
         revision = kwargs.pop('revision', None)
+        xformers_enable = kwargs.pop('xformers_enable', False)
```

Comment:
check xformers here  when xformers_enable=True,  if xformers does not exists, print hint to tell user to install xformers

Reply:
Okay, it's resolved